lib/manageiq/api/client/authentication.rb

```diff
@@ -13,6 +15,8 @@ class Authentication
         }.freeze
 
         def initialize(options = {})
+          custom_inspect_exclude(:password)
```

Comment:
This should be at the class level, not at the instance level.  I would expect that CustomInspectMixin would mixin a class method, and you could call it immediately after you include it, which keeps the logic together as well.